sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/src/Customized/CognitiveServicesAccountDeploymentResource.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core;
+using Azure.ResourceManager.CognitiveServices.Models;
+using System.Threading.Tasks;
+using System.Threading;
+
+namespace Azure.ResourceManager.CognitiveServices
+{
+    /// <summary>
+    /// CognitiveServicesAccountDeploymentResource
+    /// </summary>
+    public partial class CognitiveServicesAccountDeploymentResource
+    {
+        /// <summary>
+        /// UpdateAsync
+        /// </summary>
+        public virtual async Task<ArmOperation<CognitiveServicesAccountDeploymentResource>> UpdateAsync(WaitUntil waitUntil, CognitiveServicesAccountDeploymentData deployment, CancellationToken cancellationToken = default)
```

Comment:
Keep the same parameter name in case people are using named parameters to invoke a method.
```suggestion
        public virtual async Task<ArmOperation<CognitiveServicesAccountDeploymentResource>> UpdateAsync(WaitUntil waitUntil, CognitiveServicesAccountDeploymentData data, CancellationToken cancellationToken = default)
```

Previous signature is https://github.com/Azure/azure-sdk-for-net/blob/d96e4ea52f6078e6e09d7933c6281ff94880d435/sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/src/Generated/CognitiveServicesAccountDeploymentResource.cs#L280